system/lib/libc/musl/src/temp/__randname.c

```diff
@@ -11,6 +11,11 @@ char *__randname(char *template)
 
 	__clock_gettime(CLOCK_REALTIME, &ts);
 	r = ts.tv_nsec*65537 ^ (uintptr_t)&ts / 16 + (uintptr_t)template;
+
+	/* XXX EMSCRIPTEN: avoid repeating the same result when __clock_gettime does not change between calls. */
+	static unsigned int seed = 0;
+	r += seed++;
```

Comment:
Is `seed` the right name for this?  How about `increment` or `counter`?

Reply:
updated, counter looks good